### PR TITLE
Apollos upgrade 1.4.3

### PR DESCRIPTION
**Pull request by conrad-vanl**
### What does this PR do, or why is it needed?

Upgrades to 1.4.3 and enables prayer feature. Also re-organizes home-feed features a bit based on feedback from Liz.

@vinnyjth would you test / look this over and let me know what you think? I'd like to get prayer out in a testflight build for Liz to test this week, with goal of soft-launching it next.

## TODO:

- [ ] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [ ] Closes [tag-issues-here]
- [ ] No new warnings in tests, in storybook, and in-app
- [ ] Upload GIF(s) of iOS and Android
- [ ] Set a relevant reviewer

## REVIEW:

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

File: packages/WillowCreekApp/loadConfig.js [hunk 1]
```diff
@@ -5,31 +5,18 @@ import gql from 'graphql-tag';
 ApollosConfig.loadJs({
   FRAGMENTS: {
     ...FRAGMENTS,
-    ACTION_LIST_FEATURE_FRAGMENT: gql`
-      fragment ActionListFeatureFragment on ActionListFeature {
+    RELATED_NODE_FRAGMENT: gql`
+      fragment RelatedFeatureNodeFragment on Node {
         id
-        title
```
Comment (vinnyjth) [inline packages/WillowCreekApp/loadConfig.js:11]: Good catch. This bit was important.

**Comment by vinnyjth** [comment]
This looks good at a glance, I'll try and pull it down and run it at some point.

**Comment by conrad-vanl** [comment]
> This looks good at a glance, I'll try and pull it down and run it at some point.

Sounds good. I'm sure we'll QA it, but I was noticing what felt like instability around:

1. signing up as a new user
2. logging out
3. Random bugsnag RSODs or other errors

Some or all of the above might've been related to just the extended Bugsnag logging you put together, and I didn't have bugsnag set up correctly locally (missing keys), but wanted to call that out nonetheless

File: packages/WillowCreekApp/src/ui/ActionListCard/ActionListImage.js [hunk 2]
```diff
@@ -46,6 +46,7 @@ const ActionListImage = ({ isLoading, source, start, type }) => {
   if (isLoading) {
     return <CellImage />;
   }
+  console.warn(type, source);
```
Comment (vinnyjth) [inline packages/WillowCreekApp/src/ui/ActionListCard/ActionListImage.js:49]:
```suggestion
```
